src/wp-includes/pluggable.php

```diff
@@ -2448,16 +2448,23 @@ function wp_salt( $scheme = 'auth' ) {
 			 * https://i18n.svn.wordpress.org/<locale code>/branches/<wp version>/dist/wp-config-sample.php
 			 */
 			$duplicated_keys[ __( 'put your unique phrase here' ) ] = true;
+			$option_prime = array();
```

Comment:
I think this would be clearer with a rename. 

:1234: Applies throughout, including the suggested change I've made.

```suggestion
			$options_to_prime = array();
```